### PR TITLE
feat(policies): add tags support

### PR DESCRIPTION
**Pull request by hazedav**
## Summary
As a Lacework user I want to use the CLI to interact (CRU) with policy tags

- Add list-tags command
- Add --tag filter for policy list
- Add tag to appropriate structs

## How did you test this change?
Automated unit and integration testing

## Issue
https://lacework.atlassian.net/browse/ALLY-878

## Examples
List policies with a specific tag
```
lacework policy list --tag control:1.1 
```

List all policy tags
```
lacework policy list-tags                         
```

### CODE REVIEW

File: api/policy.go [hunk 1]
```diff
@@ -56,45 +57,61 @@ This would prevent someone from toggling something to disabled or 0 respectively
...
+	LastUpdateUser string   `json:"lastUpdateUser"`
+}
+
+func (p *Policy) HasTag(t string) bool {
+	for _, v := range p.Tags {
+		if t == v {
+			return true
+		}
+	}
+	return false
```
Comment (dmurray-lacework) [inline api/policy.go:102]: Could this be condensed```suggestion
			return  t == v
	}
```

Comment (dmurray-lacework) [inline api/policy.go:102]: could we use `ContainsStr` from go-sdk/internal/array/contains.go here?
```suggestion
return array.ContainsStr(p.Tags, t)
```

**Review by dmurray-lacework — APPROVED** [review]
Approved with one suggestion

File: api/policy.go (hunk 1 above)
Comment (hazedav) [reply api/policy.go:102]: Thanks @dmurray-lacework , admittedly I wasn't aware we had this in our extended library.

Comment (hazedav) [reply api/policy.go]: Fixed.

**Pull request merged**